torch/cuda/_dynamo_graphs.py

```diff
@@ -0,0 +1,160 @@
+import torch
+from torch.fx import GraphModule
+from torch.nn import Module
+from torch.fx.passes.backends.cudagraphs import partition_cudagraphs
```

Comment:
what does partitioning actually mean here? Looking at the function seems to indicate that it removes all non CUDA operations from a graph

Reply:
Suppose you have a program like

```
cuda op 1
cuda op 2
cuda op 3
cpu op
cuda op 4
cuda op 5
cuda op 6
```

partitioning puts cuda 1-3 and 4-6 in separate fusion groups that cuda graphs can be applied on (cuda graphs can't be used on cpu ops)